clouddrift/adapters/gdp6h.py

```diff
@@ -513,9 +509,9 @@ def to_raggedarray(
         indices=ids,
         preprocess_func=preprocess,
         name_coords=gdp.GDP_COORDS,
-        name_dims=gdp.GDP_DIMS,
         name_meta=gdp.GDP_METADATA,
         name_data=GDP_DATA,
+        dim_names=gdp.GDP_DIMS,
```

Comment:
what do you think of `name_dims` instead so it matches with `name_coords`, `name_meta`, and `name_data`.

Reply:
that makes sense, I could also swap the mapping as it should reduce the complexity of the app